operations/patch_util.go

```diff
@@ -176,6 +176,30 @@ func (p *patchParams) displayPatch(conf *ClientSettings, newPatch *patch.Patch)
 	return nil
 }
 
+func (p *patchParams) displayCommitQueuePatch(conf *ClientSettings, newPatch *patch.Patch, uiV2Url string) error {
+	patchDisp, err := getCommitQueuePatchDisplay(newPatch, p.ShowSummary, uiV2Url)
+	if err != nil {
+		return err
+	}
+
+	grip.Info("Patch successfully created.")
+	grip.Info(patchDisp)
+
+	if p.Browse {
```

Comment:
Could move this to a helper function so we can call it here and displayPatch (rather than having the same code in both places)?